corrscope/renderer.py

```diff
@@ -64,6 +65,8 @@
 # matplotlib.use() only affects pyplot. We don't use pyplot.
 import matplotlib
 import matplotlib.colors
+import matplotlib.pyplot as pyplot
+import matplotlib.patheffects as path_effects
```

Comment:
my project style is usually `from package import module`. not sure if that's better or worse than `import package.module as module`. In your case, that extra underscore is... inconsistent.

Honestly I prefer how Rust does things. I don't know how JS imports work.